src/Microsoft.DotNet.Build.CloudTestTasks/UploadToAzure.cs

```diff
@@ -1,96 +1,229 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This license shouldn't be deleted (and it should be added to all the other files). Probably drop the <copyright/summary> stuff too.